evmjit/libevmjit/Common.h

```diff
@@ -20,25 +19,30 @@ namespace jit
 using byte = uint8_t;
 using bytes = std::vector<byte>;
 using bytes_ref = std::tuple<byte const*, size_t>;
+using code_iterator = byte const*;
 
 struct NoteChannel {};	// FIXME: Use some log library?
 
 enum class ReturnCode
```

Comment:
Many reasons for no exceptions, some of them are: C/Go interface, exceptions are OS dependent , they my need to pass JIT-generated code.